pkg/blobstore/configuration/icas_blob_access_creator.go

```diff
@@ -21,7 +22,7 @@ type icasBlobAccessCreator struct {
 // provided to NewBlobAccessFromConfiguration() to construct a
 // BlobAccess that is suitable for accessing the Indirect Content
 // Addressable Storage.
-func NewICASBlobAccessCreator(grpcClientFactory grpc.ClientFactory, maximumMessageSizeBytes int) BlobAccessCreator {
+func NewICASBlobAccessCreator(terminationGroup program.Group, grpcClientFactory grpc.ClientFactory, maximumMessageSizeBytes int) BlobAccessCreator {
```

Comment:
Could you revert this change, please? This function argument isn't used.

Reply:
Whoops sorry - done.